docs/docker.rst

```diff
@@ -99,4 +99,27 @@ and enable the IPC_LOCK capability for memory registration:
 You need to specify these additional configuration options on primary and secondary workers.
 
 
+Running containers with different ports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``horovodrun`` uses SSH to launch processes on the workers. When using Docker, it may be the case that different containers
+are placed on the same host with different ports.  In such cases, you can configure your 
```

Comment:
To make the steps prominent for the reader, after "different ports" you could say, "To run containers with different ports:" and then have the following steps as an ordered list:

1. Configure your ~/.ssh/config file to assign custom host names and ports for each container
   (example)

2. Use ``hordovodrun`` directly as though each container were a separate host with its own IP
   (example)

Reply:
Good suggestion!  Updated.